### PR TITLE
Add ProfiLED page

### PR DESCRIPTION
**Pull request by IamPete1**
This adds a ProfiLED page and spits it and NeoPixles out to separate sub-pages of the serial led page.

### CODE REVIEW

File: common/source/docs/common-serial-led-ProfiLED.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.. _common-serial-led-ProfiLED:
+
+ProfiLED
+========
+
+.. image:: ../../../images/ProfiLED.jpg
+
+ProfiLED are spaced at 20mm resulting in 50 LED's per meter. The strips can be cut at any point and extended or have a new header added and plugged into a second pair of output pins. ProfiLED's require both clock and data signal. As supplied the clock is the yellow wire and green is data. Both the clock and data pins must be in the same PWM group. It is possible to share a clock pin over multiple strips in the same group. If there are spare pins it may be more convenient to setup two clock pins for that group. ProfiLED's require 5 volts and can draw a significant amount of current if all turned on at once. ProfiLED's default to white until a valid signal is received.
```
Comment (Hwurzburg) [inline common/source/docs/common-serial-led-ProfiLED.rst:8]: Pete, I think we need a diagram or two to show exactly how to connect to the autopilot in addition to the words

File: common/source/docs/common-external-leds.rst [hunk 2]
```diff
@@ -34,30 +34,24 @@ Because most users use an external :ref:`GPS and Compass module <common-position
...
-
-.. note:: Be sure the output is configures as normal PWM instead of a GPIO output. See :ref:`common-gpios`
-
-If used for notification purposes, be sure to set :ref:`NTF_LED_TYPES<NTF_LED_TYPES>` to "Neopixel" (8).
-
-.. warning:: Most WS2812 style LED and strings will operate correctly when connected to the autopilot. However, if you get intermittent or non-operation, you may need to implement one of the configurations below. This is due to the fact that the autopilot outputs swing to 3.3V but the worst case input signal high spec for the LED is 4.3V at a 5V supply. So at extremes of tolerance/manufacturing spec, you can get a combination which will not work correctly. In that case the easiest solution is to lower the LED supply as shown below.
+Serial LEDs require a whole pwm group, more information about your flight controllers PWM groups can be found on it's hardware page. Flight controllers with a IOMCU can only use the AUX outputs for connecting serial LEDs. To check that the desired outputs have been setup correctly look for the PWM types message in your GCS's messages tab.
```
Comment (Hwurzburg) [inline common/source/docs/common-external-leds.rst:42]: ...on its :ref:`hardware page<common-autopilots>`

next sentence should be a .. note ::  I think

at the end....messages tabe (see :ref:`<common-gpios>`)   I will update that page with info on the banner msgs from RCOut to GCS

formatting of above will probably be screwed up by this editor...but you get the idea

File: common/source/docs/common-serial-led-ProfiLED.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.. _common-serial-led-ProfiLED:
+
+ProfiLED
+========
+
```
Comment (Hwurzburg) [inline common/source/docs/common-serial-led-ProfiLED.rst:5]: this feature available in Copter 4.0 and later

File: common/source/docs/common-serial-led-ProfiLED.rst (hunk 1 above)
Comment (IamPete1) [reply common/source/docs/common-serial-led-ProfiLED.rst:8]: They do come with the correct header on, you just plug them in, I will try and find a picture

**Comment by Hwurzburg** [comment]
Peter, I have Googled and Googled and cant find a source for ProfiLEDs anywhere, nor ANY information about them....seem to be pretty hard to find....perhaps under a different name?

File: common/source/docs/common-serial-led-ProfiLED.rst (hunk 1 above)
Comment (IamPete1) [reply common/source/docs/common-serial-led-ProfiLED.rst]: added a pic of them connected to a cube

File: common/source/docs/common-external-leds.rst (hunk 2 above)
Comment (IamPete1) [reply common/source/docs/common-external-leds.rst]: Done

File: common/source/docs/common-serial-led-ProfiLED.rst (hunk 3 above)
Comment (IamPete1) [reply common/source/docs/common-serial-led-ProfiLED.rst:5]: Done

**Comment by IamPete1** [comment]
Also added a where to buy header with links to the Cube resellers page

**Comment by Hwurzburg** [comment]
Peter, the connection drawing is a bit hard to see whats what...maybe this?: https://www.dropbox.com/s/yzw85ctr8ni9sdq/ProfiLED%20wiring.jpg?dl=0

other than that LGTM for when the backport happens...

**Comment by IamPete1** [comment]
Added @Hwurzburg's excellent diagram and rebased, changes have also gone into copter so this will be included in the nest beta

**Comment by Hwurzburg** [comment]
merged...thanks!!

**Pull request merged**